### ISSUE TITLE
[Retail Bug Report] Gossip Frame Text Color Unreadable

### ISSUE THREAD

**Issue by Xeronel** [tukui-org/ElvUI#463]
### What is the issue you are having?

Text in gossip frames on items in the world is unreadable due to the color of the text.

Screenshot of the issue:
![image](https://user-images.githubusercontent.com/3720649/198728193-a5da4360-50c6-4662-b9b2-ec7257060362.png)

I have these options set under the "Skins" section
![image](https://user-images.githubusercontent.com/3720649/198728082-18f4dcfb-ffec-41e4-9b18-451115a4f3d5.png)

As a temporary fix I have unchecked "Gossip Frame" under the "Blizzard" list.

### What is the expected behavior?

Gossip frame text should be white or some other color that is easy to read.

### What actually happened?

Gossip frame text is dark and cannot be read.

### Suggested Solution/Workaround

Change gossip frame text to white.

### ElvUI Changelog & Troubleshooting

- [X] I have confirmed that I have read the [Changelog](https://www.tukui.org/download.php?ui=elvui&changelog).
- [X] I have verfied ElvUI is up to date by checking /estatus.
- [X] I have verfied my issue doesn't exist in the ElvUI Issue Tracker.
- [ ] I have not read step six and I am just clicking all boxes.
- [X] I have verfied my issue doesn't exist in the default UI.
- [X] I have disabled all addons except the two called 'ElvUI' and 'ElvUI OptionsUI' by using `/edebug on`? This is important in order to rule out that any of your other addons are causing the issue. If you manage to pinpoint a conflict between ElvUI and another addon, make sure you include that information.

**Comment by Caedis** [comment]
![image](https://user-images.githubusercontent.com/18713839/198814468-a621e193-cb8e-4310-82f9-6d5cd500a8d9.png)

Please do that first and check again.

Also, how are you updating ElvUI?

**Comment by Xeronel** [comment]
I'm updating using wowup but to be sure that was not the cause of the issue I have deleted the addon and reinstalled it manually.

I did disable other addons using `/dedebug on` before reporting this which is why I checked that box but here's proof along with an image of the item I'm clicking that's creating the frame in game.

![image](https://user-images.githubusercontent.com/3720649/198827815-e5f36221-1344-4ce8-b921-5520c2a21fef.png)

Aspirant's Crucible in Bastion in the Shadowlands (55.72, 84.01)
![image](https://user-images.githubusercontent.com/3720649/198828150-8bbaea17-dc37-4206-8685-205db25cf30c.png)

If this is also a gossip frame it appears that when speaking to an NPC the font color is set correctly.
![image](https://user-images.githubusercontent.com/3720649/198829272-e65e5a88-3b89-4718-ade4-fac2e47ad537.png)
